packages/legacy-workbench/src/js/angular/guides/steps/complex/enable-autocomplete/plugin.js

```diff
@@ -1,25 +1,22 @@
+const ENABLE_AUTOCOMPLETE_DEFAULT_TITLE = "guide.step-action.enable-autocomplete";
+
 PluginRegistry.add('guide.step', [
     {
-        guideBlockName: 'enable-autocomplete',
+        guideBlockName: 'click-autocomplete-checkbox',
```

Comment:
Change the step name to 'enable-autocomplete' because it actually has logic to enable autocomplete.